Legal.md

```diff
@@ -64,25 +64,18 @@ For instance, what if if(we) actually _were_ to create a non-profit foundation f
 
 Enter the [Developer's Certificate Of Origin](./starter_box/DCOO.md) (aka the DCOO).
 
-NOTE: ALL the below about the CLA needs to be replaced with information on the DCOO
-
-A CLA is just a specialized form of license. Just as the Apache License grants the right to use copyrighted software, a CLA is a license which grants the right for an individual or organization to use the work of the contributor. The contributor retains copyright over the work, but if(we) is allowed to distribute and administer the work as needed.
+The DCOO ensures that any contribution made to the project was created by the contributor and that it is not under any other copyright.  It also ensures that the contributor must certify that they are allowed to contribute the code and is under the appropriate open source license.
```

Comment:
Grammar nit--I think this is better:
"contribute the code and that the code is under"